src/xcode/ENA/ENA/Source/Scenes/Settings/SettingsViewController.swift

```diff
@@ -134,18 +134,20 @@ final class SettingsViewController: UITableViewController {
 	private func notificationSettings() {
 		let currentCenter = UNUserNotificationCenter.current()
 
-		currentCenter.requestAuthorization(options: [.alert, .sound, .badge]) { [weak self] granted, error in
+		currentCenter.getNotificationSettings { [weak self] (settings) in
```

Comment:
No need for braces around the parameter.

Reply:
Fixed